docs/rules/eqeqeq.md

```diff
@@ -67,6 +67,13 @@ foo === null
 
 ```
 
+This rule optionally takes a second argument which should be an object with the following supported properties:
+
+* `"null"`: Customize how this rule treats `null` literals. Possible values:
+    * `always` (default) - Always use type-safe equality.
+    * `never` - Never use type-safe equality with `null`.
```

Comment:
"Type-safe equality" is technically accurate, but everywhere else we seem to just be using the operator everywhere. To be consistent, could we have the always section say something like "Always use `===` or `!==`", and similarly for the never section? Thanks!